host-ext/crouton/window.js

```diff
@@ -262,8 +269,10 @@ function handleFocusBlur(evt) {
 }
 
 /* Start in full screen */
-chrome.windows.update(chrome.windows.WINDOW_ID_CURRENT,
-                      {'state': "fullscreen"}, function(win) {})
+if (location.search.search(/[&?]mode=f(&|$)/i) != -1) {
```

Comment:
Wouldn't it be better to do the argument parsing a bit earlier then? (the parameter extraction code and global variable assignments can be moved here, and force a `setHiDPI` and `setDebug` in the listener below)